CHANGELOG.rst

```diff
@@ -3,6 +3,12 @@ Changelog
 
 in development
 --------------
+Added
```

Comment:
Looks like we have 2 `Added` blocks snow.
Please move the Changelog record under the existing `Added` section.

Reply:
Done.